media-video/tsmuxer/tsmuxer-2.6.16_p20220706.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
I'm sorry it's taken so long to review this. Can you bump up to EAPI=8?

Reply:
Done